packages/office-ui-fabric-react/src/components/Calendar/Calendar.test.tsx

```diff
@@ -173,6 +173,7 @@ describe('Calendar', () => {
           dateRangeType={ DateRangeType.Week }
           autoNavigateOnSelection={ true }
           onSelectDate={ onSelectDate() }
+          className="CalendarTestClass"
```

Comment:
Travis is failing due to the double quote marks. Change all `"` to `'` for it to pass.